README.md

```diff
@@ -36,11 +33,7 @@ https://example.com/asfjaflnadsesljarmdkasdjasd. Just add a new line to
 `shortlinks.csv` with `my-shortlink` as the `keyword` and your complex
 URL as the `destination_url`.
 
-Shortlinks are **updated nightly** as part of the [`update_shortlinks`
-task](https://github.com/hyphacoop/worker-coop-scripts#update_shortlinks)
-in the [`hyphacoop/worker-coop-scripts` repo][scripts]. A [manual
-update][manual-update] can also be run on-demand, which is helpful
-for testing.
+Shortlinks are active as soon as **commited to maters**. Github may cache results of a short priod of time.
```

Comment:
Spellcheck

Reply:
Also they have a 10 min delay bc of @ASoTNetworks's caching. Where is that code and documented behaviour?

Reply:
I have no knowledge of this change.

Reply:
The change is on line 6 here https://github.com/hyphacoop/shortlinks-site/blob/master/redir.php#L6

Instead of the `https://raw.githubusercontent.com/hyphacoop/shortlinks/master/shortlinks.csv` it is changed to `shortlinks.csv`

Crontab that will wget the file every 10 minutes
`*/10 * * * * wget https://raw.githubusercontent.com/hyphacoop/shortlinks/master/shortlinks.csv -O /var/www/link.hypha.coop/shortlinks.csv`

Reply:
@ASoTNetworks https://github.com/hyphacoop/shortlinks-site/pull/3/files#r427145236

Reply:
> The change is on line 6 here https://github.com/hyphacoop/shortlinks-site/blob/master/redir.php#L6
> 
> Instead of the `https://raw.githubusercontent.com/hyphacoop/shortlinks/master/shortlinks.csv` it is changed to `shortlinks.csv`
> 
> Crontab that will wget the file every 10 minutes
> `*/10 * * * * wget https://raw.githubusercontent.com/hyphacoop/shortlinks/master/shortlinks.csv -O /var/www/link.hypha.coop/shortlinks.csv`

Why not just keep all this internally

have the script cache the file when its run instead of cron job?